crowd_anki/github/github_importer.py

```diff
@@ -8,6 +8,16 @@
 
 BRANCH_NAME = "master"
 
+
+def get_repository_name(repository_url):
+    repo_url_path = urlparse(repository_url).path
```

Comment:
what does using urlparse give us?

Reply:
true, not much. I thought it could help in cases when the URL contains query parameters, but nothing in github seems to.